airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-package io.airbyte.integrations.base;
+package io.airbyte.integrations.standardtest.destination;
```

Comment:
maybe add an `_`

Reply:
okay. this is going to be hard to change now because of the other branches stacked on top of it. once they are all merged in, i will come back and do this.